utils/lpac/files/lpac.sh

```diff
@@ -8,26 +8,29 @@ APDU_DEBUG="$(uci_get lpac global apdu_debug 0)"
 HTTP_BACKEND="$(uci_get lpac global http_backend curl)"
 HTTP_DEBUG="$(uci_get lpac global http_debug 0)"
 
-export LPAC_HTTP="$HTTP_BACKEND"
+function export_if_not_in_env {
+    eval "value=\${$1}"
+    if [ -z "$value" ]; then
+        export "$1=$2"
+    fi
+}
+
+export_if_not_in_env LPAC_HTTP "$HTTP_BACKEND"
 if [ "$HTTP_DEBUG" -eq 1 ]; then
-    export LIBEUICC_DEBUG_HTTP="1"
+    export_if_not_in_env LIBEUICC_DEBUG_HTTP "1"
```

Comment:
https://github.com/openwrt/packages/pull/25046#issuecomment-2385613465
> The problem with your handling of the debug options, as I see it, is that you have to interpret their values, which our scripts shouldn't do anyway.

Not addressed.
See the alternative approach above, again.

Reply:
So you mean to not set the debug env variables if they are not enabled? This would suffice if we remove UCI handling from them, which i am fine with.

Reply:
Right now, the UCI handling is in place, so we could (and should) rely on that instead of trying to be smart and interfere with lpac's logic in this script.